src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolderDescriptor.java

```diff
@@ -41,6 +41,40 @@ public String getDisplayName() {
     }
 
     /**
+     * Needed if it wants Folders are categorized in Jenkins 2.x.
+     *
+     * TODO: Override when the baseline is upgraded to 2.x
+     * TODO: Replace to {@code NestedProjectsCategory.ID}
```

Comment:
> NestedProjectsCategory will be provide by core though it can be provided by this plugin

It is fine to _initially_ keep the category in core, so long as it is `@Restricted` so we retain the option of moving it into `cloudbees-folder` later.